code/modules/power/supermatter/supermatter.dm

```diff
@@ -27,126 +14,71 @@
 #define DETONATION_HALLUCINATION 600
 
 
-#define WARNING_DELAY 30			//seconds between warnings.
+#define WARNING_DELAY 30 		//seconds between warnings.
 
 /obj/machinery/power/supermatter
-	name = "Supermatter"
-	desc = "A strangely translucent and iridescent crystal. \red You get headaches just from looking at it."
+	name = "supermatter shard"
+	desc = "A strangely translucent and iridescent crystal that looks like it used to be part of a larger structure. <span class='danger'>You get headaches just from looking at it.</span>"
```

Comment:
Not changing the sprite, so the previous description should be used and not the new one, also needs it's name changed.